.github/workflows/release.yml

```diff
@@ -0,0 +1,117 @@
+    if: ${{ needs.check_skip.outputs.should_skip != 'true' }}
+    steps:
+      - name: Checkout repository and submodules
+        uses: nschloe/action-cached-lfs-checkout@v1
+        with:
+          submodules: recursive
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            i8n:
+              - 'include/locales/*.h'
+      - id: get-languages
+        run: |
+          if [ "${{ steps.filter.outputs.i8n }}" == "true" -o "${{ github.head_ref == 'master' }}" == "true" -o "${{ github.base_ref == 'master' }}" == "true" ]; then
+            echo "languages_matrix=$(python3 .github/getWorkflowMatrix.py all_languages)" >> $GITHUB_OUTPUT
+          else
+            echo "languages_matrix=$(python3 .github/getWorkflowMatrix.py default_language)" >> $GITHUB_OUTPUT
+          fi
+      - id: get-models
+        run: echo "models_matrix=$(python3 .github/getWorkflowMatrix.py all_models)" >> $GITHUB_OUTPUT
+    outputs:
+      languages_matrix: ${{ steps.get-languages.outputs.languages_matrix }}
+      models_matrix: ${{ steps.get-models.outputs.models_matrix }}
+
+  build-OSW:
+    needs: Find-packages
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ${{ fromJson(needs.Find-packages.outputs.languages_matrix) }}
+        model: ${{ fromJson(needs.Find-packages.outputs.models_matrix) }}
+        build-configuration: [debug, release]
+    steps:
+      - name: Checkout repository and submodules
+        uses: nschloe/action-cached-lfs-checkout@v1
+        with:
+          submodules: recursive
+      - name: Cache pip
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ runner.os }}
+      - name: Cache PlatformIO
+        uses: actions/cache@v4
+        with:
+          path: ~/.platformio
+          key: platformio-${{ runner.os }}
+      - name: Install swig
+        run: sudo apt-get update && sudo apt-get -y install swig
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+      - name: Install PlatformIO
+        run: python -m pip install --upgrade pip && pip install --upgrade platformio
+      - name: Rename config
+        run: mv include/config.h.example include/config.h
+      - name: Compile language ${{ matrix.language }} model ${{ matrix.model }}
+        run: python3 .github/buildFirmware.py -l "${{ matrix.language }}" -m "${{ matrix.model }}" -b "${{ matrix.build-configuration }}"
+      - name: Upload firmware artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: firmwares
+          path: |
+            *.bin
+  publish:
+    needs: build-OSW
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      # create the GitHub release
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1.1.4
```

Comment:
While this runs on the tag, what will the pre-filled release-text be?

Reply:
Did you ok ? See my new answer with the test picture.